add documentation for jstor parser (#319)
JabRef/user-documentation: en/collect/import-using-online-bibliographic-database.md

* _a smith or a jones_: search for references with either author "smith" or author "jones"
* _a smith and not t processor_: search for author "smith" and omit references with "processor" in the title

### Jstor

[Jstor](https://jstor.org) is an online databse with access to more than 12 million journal articles, books and sources in 75 disciplines. [About](https://about.jstor.org/)

> It is possible to limit the search by adding a field name to the search, as field:"text". The supported fields area:

Reviewer: ```suggestion
It is possible to limit the search by adding a field name to the search, such as `field:"text"`. The supported fields are:
```

> * ti: The title of the article
> * au: a author of the article
> * pt: publication title

Reviewer: ```suggestion
* `ti`: The title of the article
* `au`: a author of the article
* `pt`: publication title
```
